tests/support-us.spec.ts

```diff
@@ -1,12 +1,11 @@
 import { expect, test } from '@playwright/test';
 
-test.fail();
-
 test.beforeEach(async ({ page }) => {
 	await page.goto('/support-us');
 });
 
 test('support-us page screenshot', async ({ page }) => {
+	await page.getByRole('contentinfo').scrollIntoViewIfNeeded();
```

Comment:
Ok, this is the probably the line I am missing in about-us.spec.ts

Reply:
Yeah, because the images are all loaded lazy (only when they are in view) they would be missing in the screenshot. We didn't have this in the `SvelteKit` implementation.
When we later update the tests/screenshots we should be able to clean this up :)

See https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading for some details about lazy loading.